### DISCUSSION TITLE
1.2.3 Christmas Edition stable

### DISCUSSION THREAD

**Discussion by trueromanus** [anilibria/anilibria-winmaclinux#107] [Анонсы]
Анонс изменений в версии 1.2.3:
- Исправлена ошибка возврата в значения окна с кастомным тулбаром из видеоплеера
- Исправлена ошибка синхронизации релизов
- Исправлена ошибка когда отфильтрованные поля неправильно перекрывались в редакторе тем
- Добавлены стрелки в горизонтальных списках на странице Моя Анилибрия
- Добавлены стрелки в горизонтальных списках на странице Каталог релизов в карточке релиза
- Возврат на настоящую стартовую страницу по настройке Для стартовой страницы из плеера
- На windows плеер если доступно будут использоваться кодеки (DirectX11, DirectX Video Acceleration, CUDA) из системы
- В опции "Размер буфера" добавилось больше вариантов
- Случайный релиз по избранному, по текущему фильтру и по просматриваемому
- Отображение названий серий в списке серий
- Добавлен цвет рамки заголовка регионов на странице Моя Анилибрия
- Настройка Включить видео прокси (пока в виде эксперимента)

Всех с наступающим новым годом!

![image](https://user-images.githubusercontent.com/9040186/207928564-e1f17e5b-c463-4f45-82a1-5ab71f4021ce.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/anilibria/anilibria-winmaclinux/releases/tag/1.2.3'>1.2.3 Christmas Edition stable</a>.</em>

**Comment by andrey666x** [comment]
dpkg-deb: ошибка: в архиве «anilibria_1.2.3_amd64.deb» используется неизвестное метод сжатия для «control.tar.zst», прекращаем
dpkg: ошибка при обработке архива anilibria_1.2.3_amd64.deb (--install):
 dpkg-deb --control subprocess returned error exit status 2

**Reply by trueromanus** [reply]
Очень вероятно что у Вас не самая последняя версия dpkg-deb или его зависимостей. Пакеты собираются на самой свежей ubuntu. Уточните пожалуйста свой дистрибутив и его версию.

**Reply by andrey666x** [reply]
debian 11 stable. попробую  поставить dpkg из backports.

**Reply by trueromanus** [reply]
Тогда попробуйте также поставить пакеты `libzstd1` и `zstd` или поставить [отсюда](https://packages.ubuntu.com/focal-updates/dpkg), не факт что поможет но как вариант можно попробовать.

**Comment by MrZef1rka** [comment]
![image](https://user-images.githubusercontent.com/108629989/210797096-88630170-d1be-4106-b757-0a166a8bf444.png)
здравствуйте, интересует момент, что при открытии серии, изображение уползает вниз, в предыдущих версиях такого не наблюдал, конечно при двойном клике, оно подстраивается под монитор, но всё же.
(скрин сделан всего рабочего пространства монитора)
Ещё при запуске, панель задач просто исчезает. 
У меня сложилось впечатление, что приложение не подстраивается под разрешение моего монитора, как например, браузер. Тоесть если я перетаскиваю его на второй монитор, то получаем картинку, что приложение и на первом мониторе и на втором.
Конечно, можно руками уменьшить размер окна приложения, чтобы такого не было, но было бы не плохо, если бы оно могло подстраиваться самостоятельно.
![image](https://user-images.githubusercontent.com/108629989/210799449-b85e3c21-8d45-432b-8bbc-c7bef6243a6a.png)
Так же напрягает,что приложение часто делает перемотку кадров вперед само по себе(секунды 4-5), приходится отматывать назад.

**Reply by trueromanus** [reply]
Насчет первой проблемы: можете отключить "Кастомный тулбар" в настройках.
Насчет второй проблемы: Есть версия с плеером VLC https://github.com/anilibria/anilibria-winmaclinux/releases/tag/1.2.3vlc

**Reply by MrZef1rka** [reply]
Спасибо за помощь!
